### PR TITLE
SW-2771 Add locale to user details

### PR DESCRIPTION
**Pull request by sgrimm**
Update the `/api/v1/users` payloads to include a `locale` field.

### CODE REVIEW

**Comment by sgrimm** [comment]
Current dependencies on/for this PR:
* main
  * **PR #913** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/913" target="_blank">Graphite</a>  👈
    * **PR #914** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/914" target="_blank">Graphite</a> 
      * **PR #915** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/915" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/913?utm_source=stack-comment).

File: src/main/kotlin/com/terraformation/backend/customer/api/UsersController.kt [hunk 1]
```diff
@@ -13,6 +13,7 @@ import com.terraformation.backend.db.default_schema.UserId
 import io.swagger.v3.oas.annotations.Operation
 import io.swagger.v3.oas.annotations.media.Schema
 import java.time.ZoneId
+import java.util.*
```
Comment (karthikbtf) [inline src/main/kotlin/com/terraformation/backend/customer/api/UsersController.kt:16]: .Locale sufficient .. and in other new import locations?

File: src/main/kotlin/com/terraformation/backend/db/LocaleConverter.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  /**
+   * Converts a time zone name to a ZoneId.
+   *
+   * @throws DateTimeException The zone name wasn't found in the list of zones recognized by the
+   *   java.time package.
+   */
+  override fun from(databaseObject: String?): Locale? =
+      databaseObject?.let { Locale.forLanguageTag(it) }
+
+  override fun to(userObject: Locale?): String? = userObject?.toLanguageTag()
```
Comment (karthikbtf) [inline src/main/kotlin/com/terraformation/backend/db/LocaleConverter.kt]: `localeObject`?

**Review by karthikbtf — APPROVED** [review]
lgtm, a few comments

File: src/main/kotlin/com/terraformation/backend/customer/api/UsersController.kt (hunk 1 above)
Comment (sgrimm) [reply src/main/kotlin/com/terraformation/backend/customer/api/UsersController.kt:16]: Right now we allow wildcard imports of a couple specific packages, one of them being `java.util`. You'll find `import java.util.*` in a bunch of places in the code. No specific reason, though, other than that it's the default in IntelliJ. I'm going to leave this in place for this PR, but make a separate one to try removing that exception so we can see what it looks like; I agree it's inconsistent.

Comment (karthikbtf) [reply src/main/kotlin/com/terraformation/backend/customer/api/UsersController.kt:16]: Got it, sounds good to leave it as-is for now.

Comment (sgrimm) [reply src/main/kotlin/com/terraformation/backend/customer/api/UsersController.kt:16]: Just filed SW-2789 for this so I don't forget.

**Pull request merged**